lib/screenplay/config/s3_fetch.ex

```diff
@@ -35,12 +49,35 @@ defmodule Screenplay.Config.S3Fetch do
   end
 
   defp config_path_for_environment(file_spec) do
-    base_path = "screenplay/#{Application.get_env(:screenplay, :environment_name, "dev")}"
```

Comment:
This default actually wasn't working because `Application.get_env/2` only returns the default when the key doesn't exist. Because the key was being set to `nil` in the case where the env var wasn't set, the config key technically still exists at runtime, just with a `nil` value. I made a change to set the default within `runtime.exs` instead